packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -95,6 +95,7 @@ Future<XcodeBuildResult> buildXcodeProject({
 
   for (final IOSMigrator migrator in migrators) {
     if (!migrator.migrate()) {
+      UsageEvent('ios-migration', '${migrator.usageEventAction}-failure').send();
```

Comment:
I think this would be a bit easier to navigate in GA as:
```dart
UsageEvent('ios-migration', migrator.usageEventAction, label: 'failure').send();
```